index.html

```diff
@@ -74,6 +70,10 @@
     </div>
   </footer>
 
+  <div id="add-project-form">
```

Comment:
didn't change this part much except

1. moved this block down
2. changed it from `<section>` to `<div>`